libtransmission/utils.cc

```diff
@@ -193,6 +196,15 @@ bool tr_file_save(std::string_view filename, std::string_view contents, tr_error
     {
         return false;
     }
+#ifndef _WIN32
+    // set file mode per settings umask()
+    else
+    {
+	mode_t val = umask(0);
```

Comment:
Please reformat your code, we use spaces instead of tabs.

You can run `./code_style.sh check` from project directory to check and `./code_style.sh fix` to fix automatically according project's clang-format rules.

Reply:
reformatted with clang-format, then checked with ./code-style.sh